### ISSUE TITLE
Handling Unit Editor w/ drag&drop action behavior

### ISSUE THREAD

**Issue by metas-mk** [metasfresh/metasfresh-webui-frontend#538]
### Type of issue
Feature request
### Current behavior
For Transform action: Currently, the user must select an HU Line, then press the Action drop-down list, select an action, fill in different Parameters, and press start.
### Expected behavior
What about having the possibility to drag an HU element onto another HU element or on an empty space. The system can then interpret what the user wants to do and automatically show the Parameters that have to be filled.
WDYT?

**Comment by teosarca** [comment]
also, in parallel we shall define the backend api

**Comment by damianprzygodzki** [comment]
Please, ping us when API will be ready.

**Comment by teosarca** [comment]
required also for #580

**Comment by damianprzygodzki** [comment]
Please rethink with having in mind drag&drop between manufacturing 'editor' and hu editor.

**Comment by damianprzygodzki** [comment]
@teosarca if we choose another way to solve that, will this issue be proceed?

**Comment by teosarca** [comment]
@damianprzygodzki  actually i still believe we need this feature, but just that it's low prio for now, so we postpone it.

**Comment by teosarca** [comment]
closing it for now because currently it's not in our scope anymore.
